prow/config/config.go

```diff
@@ -1830,6 +1868,14 @@ func validateTriggering(job Presubmit) error {
 	return nil
 }
 
+func validateRerunAuthConfig(rac *prowapi.RerunAuthConfig) error {
+	if rac == nil {
+		return nil
```

Comment:
Nit: You could include this in `rac.Validate()`.

Reply:
If `rac == nil`, `rac.Validate()` will fail right? Also, I am trying to be consistent in pattern with the other validation methods that act on pointers (e.g.  `validatePodSpec`, `ValidatePipelineRunSpec`, `validateDecoration`, etc.)

Reply:
Go actually allows this! In particular, functions with pointer receiver types can be safely called if the receiver is nil so long as the function does not unsafely dereference the receiver. I made a quick example: https://play.golang.org/p/MFIl_QtjX_N

If it is always considered valid for this struct to be nil then I'd include it in the validation check even if it isn't consistent with the rest of the validation functions (perhaps some of them should be doing the same). It doesn't really matter though so choose what makes the most sense to you.

Reply:
Nice! I was not aware of this feature. Thanks, I will update.

Reply:
Updated.